### ISSUE TITLE
blocking webrequest policy

### ISSUE THREAD

**Issue by tbodt** [w3engine/w3engine#1]
It's been announced that the blocking webrequest API will be available behind a policy flag. As such the fork would only need a one line code change to enable the API by default.

**Comment by w3engine** [comment]
Yes. But they could change this later, this is why a dedicated fork is a must. Google is not known very positively for enterprise support either that the flag relies on.

**Comment by tbodt** [comment]
>Google is now known very positively for enterprise support either that the flag relies on.

sorry i'm not sure what you mean by that? maybe there's a word missing?

**Comment by w3engine** [comment]
I've edited it.
